stdlib/format.mli

```diff
@@ -376,7 +376,26 @@ val set_max_indent : int -> unit
 (** [pp_set_max_indent ppf d] sets the maximum indentation limit of lines
   to [d] (in characters):
   once this limit is reached, new pretty-printing boxes are rejected to the
-  left, if they do not fit on the current line.
+  left, unless the enclosing box fully fits on the current line.
+  As an illustration,
+  {[ set_margin 10; set_max_indent 5; printf "@[123456@[7@]89A@]@." ]}
+  yields
+  {[
+    123456
+    789A
+  ]}
+  because the nested box ["@[7@]"] is opened after the maximum indentation
+  limit ([7>5]) and its parent box does not fit on the current line.
+  Contrarily, either decreasing the length of the parent box to make it fit
```

Comment:
Not sure about "Contrarily" here (I do not feel a big opposition with what was said before). Maybe we could just go with "Either" (I agree I am nitpicking)

Reply:
Good point, I removed the `contrarily`.